### ISSUE TITLE
Reduce gap between treenods in flutter_simple_treeview

### ISSUE THREAD

**Issue by Dooardarshan** [google/flutter.widgets#438]
i was wondering is there a way to Reduce the gap between treenods in flutter_simple_treeview. i takes so much space

**Comment by ramanjitsingh-hub** [comment]
Yes, you can reduce the gap between tree nodes in flutter_simple_treeview by using the  `horizontalGap` and `verticalGap ` properties. These properties control the horizontal and vertical space between nodes in the tree, respectively.

Here is an example of how you can use these properties to reduce the gap between tree nodes:

```
SimpleTreeView(
  horizontalGap: 5,
  verticalGap: 5,
  children: [
    // Tree nodes go here
  ],
),

```
In this example, the horizontalGap and verticalGap properties are set to 5, which will reduce the gap between the tree nodes by that amount. You can adjust these values to achieve the desired gap size in your tree.

**Comment by bambinoua** [comment]
@ramanjitsingh-hub what `SimpleTreeView` was you talking about? Where is this class??
